gnovm/stdlibs/std/params_test.go

```diff
@@ -0,0 +1,40 @@
+package std
+
+import (
+	"testing"
+)
+
+func TestValidate(t *testing.T) {
+	tests := []struct {
+		prefix  string
+		key     string
+		kind    string
+		wantErr bool
+	}{
+		// Valid cases
+		{"module", "valid_key.string", "string", false},
```

Comment:
these dots don't matter, so maybe rename or rename e.g. valid_key_string for clarity.